### PR TITLE
Perception: Save manually calibrated camera parameters

### PR DESCRIPTION
**Pull request by techoe**
Note: Temporarily camera parameters are saved by streaming. Please let me know if we have save_yaml function.

### CODE REVIEW

File: modules/perception/camera/tools/offline/visualizer.cc [hunk 1]
```diff
@@ -383,6 +386,244 @@ bool Visualizer::reset_key() {
   return true;
 }
 
+double Visualizer::regularize_angle(const double &radian_angle) {
```
Comment (xiaoxq) [inline modules/perception/camera/tools/offline/visualizer.cc]: Simply pass value of primitive types, instead of reference. Plz also fix others.

Comment (techoe) [reply modules/perception/camera/tools/offline/visualizer.cc]: I think reference consumes less memory than copying double type.

Comment (xiaoxq) [reply modules/perception/camera/tools/offline/visualizer.cc]: It takes as long as a pointer, so generally there is no benefit. Mordern compiler could use register to store the scala types which even gives better performance.

Comment (techoe) [reply modules/perception/camera/tools/offline/visualizer.cc]: OK. I will change double reference to call by value.

Comment (techoe) [reply modules/perception/camera/tools/offline/visualizer.cc]: I updated the code to have to call by value of double type!

**Pull request merged**